structured_external_storage_all/models/ir_attachment.py

```diff
@@ -1,8 +1,48 @@
 # -*- coding: utf-8 -*-
 # © 2018 Sunflower IT (http://sunflowerweb.nl)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from odoo import models, fields, api, _
+import logging
+import functools
+import datetime
+import dateutil.relativedelta as relativedelta
+from odoo import api, models, tools
 from odoo.tools.safe_eval import safe_eval
+from werkzeug import urls
+
+_logger = logging.getLogger(__name__)
+try:
+    from jinja2.sandbox import SandboxedEnvironment
+    mako_template_env = SandboxedEnvironment(
+        block_start_string="<%",
+        block_end_string="%>",
+        variable_start_string="${",
+        variable_end_string="}",
+        comment_start_string="<%doc>",
+        comment_end_string="</%doc>",
+        line_statement_prefix="%",
+        line_comment_prefix="##",
+        trim_blocks=True,               # do not output newline after blocks
+        autoescape=True,                # XML/HTML automatic escaping
+    )
```

Comment:
I am thinking that we can simplify this by just using the [Jinja2 defaults](http://jinja.pocoo.org/docs/2.10/api/#high-level-api). The above code tries to use Jinja2 to render Mako templates, but for us we are only using `${object.something}` anyway, so we don't care about the others and for variables we can just as well use the default Jinja2 version: `{{object.something}}`.